analyses/snv-callers/compare_snv_callers_plots.Rmd

```diff
@@ -157,27 +227,49 @@ callers_per_mutation %>%
 
 ```{r}
 # Save this plot
-ggplot2::ggsave(file.path(consensus_plots_dir, "upset_median_vaf_plot.png"))
+ggplot2::ggsave(file.path(plots_dir, "upset_median_vaf_plot.png"))
 ```
 
 ## Base Changes 
 
-Set up the summarized form of the base change information. 
+Summarize base change information per caller. 
 
 ```{r}
 # Summarize by the number of times each base change shows up in each category.
-perc_change_df <- vaf_df %>%
+perc_change_df <- all_caller %>% 
+  dplyr::mutate(base_change = paste0(Allele, ">", Reference_Allele)) %>%
+  # Case when only works for df not sql, so we will convert now
+  as.data.frame() %>%
+  dplyr::mutate(
+  # From the base_change variable, summarize insertions, deletions, and
+  # changes that are more than one base into their own groups.
+  change = dplyr::case_when(
+    grepl("^-", base_change) ~ "ins",
+    grepl("-$", base_change) ~ "del",
+    nchar(base_change) > 3 ~ "long_change",
+    TRUE ~ base_change
+  )) %>% 
+  dplyr::select(-Allele, -Reference_Allele, -Chromosome, -base_change, 
```

Comment:
Drop most of these columns before pulling from the database (before `as.data.frame()`) to save memory (if we are still pulling from the db here, and not much earlier).